plugin/src/main/java/org/jenkinsci/plugins/structs/describable/DescribableModel.java

```diff
@@ -267,12 +271,20 @@ public String getDisplayName() {
      * or {@link Class#getSimpleName} may be used in case the argument type is {@link Describable}
      * and only one subtype is registered (as a {@link Descriptor}) with that simple name.
      */
-    public T instantiate(Map<String,?> arguments) throws Exception {
+    public T instantiate(Map<String,?> arguments) throws IllegalArgumentException {
+        return instantiate(arguments, new LogTaskListener(LOGGER, Level.FINE));
+    }
+
+    public T instantiate(Map<String,?> arguments, TaskListener listener) throws IllegalArgumentException {
```

Comment:
You should copy the Javadoc from the other overload, add a description for the new parameter, and probably deprecate the old method and replace its Javadoc with `@Deprecated Use {@link #instantiate(Map, TaskListener)} instead` (or whatever the right syntax is). Ditto for the new overloads of `instantiate` in `UninstantiatedDescribable`.

Reply:
> copy the Javadoc from the other overload

For what it is worth I tend to _move_ Javadoc to the sole nondeprecated variant (leaving behind only a `@deprecated` tag), so there is no duplication.